src/components/shared/MoreTable.vue

```diff
@@ -246,28 +246,28 @@ Licensed under the Elastic License 2.0. */
     cancel(row);
   }
   function isEditable(row: any) {
-    if (props.editableAccess) {
-      if (row.userRoles) {
-        return (
-          (row.userRoles.some((r: StudyRole) =>
-            props.editAccessRoles.includes(r),
-          ) &&
-            row.status === StudyStatus.Draft) ||
-          (row.userRoles.some((r: StudyRole) =>
-            props.editAccessRoles.includes(r),
-          ) &&
-            row.status === StudyStatus.Paused) ||
-          (row.userRoles.some((r: StudyRole) =>
-            props.editAccessRoles.includes(r),
-          ) &&
-            row.status === StudyStatus.PausedPreview)
-        );
-      } else {
-        return props.editable(row);
-      }
-    } else {
+    if (!props.editableAccess) {
       return false;
     }
+
+    if (row.userRoles) {
+      return (
+        (row.userRoles.some((r: StudyRole) =>
+          props.editAccessRoles.includes(r),
+        ) &&
+          row.status === StudyStatus.Draft) ||
+        (row.userRoles.some((r: StudyRole) =>
+          props.editAccessRoles.includes(r),
+        ) &&
+          row.status === StudyStatus.Paused) ||
+        (row.userRoles.some((r: StudyRole) =>
+          props.editAccessRoles.includes(r),
+        ) &&
+          row.status === StudyStatus.PausedPreview)
```

Comment:
This should be equivalent, I guess:
```suggestion
        row.userRoles.some((r: StudyRole) =>
          props.editAccessRoles.includes(r),
        ) &&
          [StudyStatus.Draft, StudyStatus.Paused, StudyStatus.PausedPreview].contains(row.status)
```

Reply:
Elegant, thanks for the tip! I just replaced "contains()" with the js like "includes()".